Assets/MRTK/Core/Definitions/Devices/MixedRealityInteractionMapping.cs

```diff
@@ -101,6 +101,25 @@ public MixedRealityInteractionMapping(MixedRealityInteractionMapping mixedRealit
                    legacyInput.InvertXAxis,
                    legacyInput.InvertYAxis) { }
 
+        public MixedRealityInteractionMapping(uint id, MixedRealityInputActionMapping mixedRealityInputActionMapping)
+            : this(id,
+                   mixedRealityInputActionMapping.Description,
+                   mixedRealityInputActionMapping.AxisType,
+                   mixedRealityInputActionMapping.InputType,
+                   mixedRealityInputActionMapping.InputAction) { }
+
+        public MixedRealityInteractionMapping(uint id, MixedRealityInputActionMapping mixedRealityInputActionMapping, MixedRealityInteractionMappingLegacyInput legacyInput)
```

Comment:
nit: more readable to place method args on their own lines. :smile:

Reply:
there are some other files with really long method signatures that can benefit from going multi-line, too

Reply:
I'll also align the braces. Any opinions on

```c#
public Foo(int Bar) { }
```

vs

```c#
public Foo(int Bar)
{ }
```

vs

```c#
public Foo(int Bar)
{
}
```

I'm leaning 2 or 3.

Reply:
2 is my favorite

Reply:
Updated the definitions/controllers with empty constructors to number 2